### PR TITLE
Updating go.mod and alpine

### PR DESCRIPTION
**Pull request by stephpalis**

### CODE REVIEW

**Comment by blkperl** [comment]
Can you put commit 7868360 in a different PR? Ideally we would like to squash the PR but we can't do that with the second author.

**Review by blkperl — APPROVED** [review]

File: pkg/tombstone/tombstone.go [hunk 1]
```diff
@@ -113,24 +114,24 @@ func Read(graveyard, name string) (*Tombstone, error) {
...
-type EventHandler func(fsnotify.Event)
+type EventHandler func(string, string, fsnotify.Op)
 
 // LoggingEventHandler is an example EventHandler that logs fsnotify events
-func LoggingEventHandler(event fsnotify.Event) {
-	if event.Op&fsnotify.Create == fsnotify.Create {
-		log.Info("Tombstone Watch: file created:", "name", event.Name)
+func LoggingEventHandler(graveyard string, tombstone string, op fsnotify.Op) {
+	if op&fsnotify.Create == fsnotify.Create {
+		log.Printf("Tombstone Watch: file created: %s/%s\n", graveyard, tombstone)
```
Comment (askore) [inline pkg/tombstone/tombstone.go:122]: Why are we not using kubelog here?

Comment (blkperl) [reply pkg/tombstone/tombstone.go:122]: Oh, right @stephpalis you are going to need to change this patch to incorporate @askore's changes in the previous pr.

Comment (stephpalis) [reply pkg/tombstone/tombstone.go:122]: New PR is up undoing this change and putting that commit in a different PR: https://github.com/acquia/kubexit/pull/3

Will cleanup this PR once that is approved.

**Review by askore — APPROVED** [review]

**Pull request merged**